Wikipedia/Code/wikipedia-namespaces/test.json

```diff
@@ -0,0 +1,49 @@
+{
+  "languagecode": {
+    "en": {
```

Comment:
i think this "en" should be "test" as well?

Reply:
PR to update the test file when the others are updated:
https://github.com/wikimedia/wikipedia-ios/pull/3354